go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/AdamSLevy/jsonrpc2 v2.0.0+incompatible
 	github.com/AdamSLevy/jsonrpc2/v11 v11.3.2
+	github.com/AdamSLevy/jsonrpc2/v13 v13.0.1
```

Comment:
why is /v11 still in go.mod? if it's not used anywhere we should tidy it

Reply:
I don't know. I can try a tidy and see if that removes it :thinking: